src/Framework/CompatibilitySuppressions.xml

```diff
@@ -1,7 +1,70 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <!-- PKV004 for netstandard2.0-supporting TFs that we do not have runtime assemblies for.
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
```

Comment:
Is this just all stuff that popped up due to bugfixes in the compat analyzer?

Reply:
yes, exactly.